### PR TITLE
Sala red patch

### PR DESCRIPTION
**Pull request by lsala**
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/toolbox/pulls) for the same change?
- [ ] Have you fixed a Bug ?
- [ ] Have you added the documentation of the example or benchmark ?
- [ ] Have you successfully run the example or benchmark?
- [ ] Have you documented the results?
- [ ] Have you added the images to the documentation?
- [ ] Have you added the vtkjs files for 3D visualization in the documentation?
- [ ] Have you added the vtkjs files to a Data Management Plateform _e.g_ https://girder.math.unistra.fr

-----

@prudhomm I have updated the README with command file and data in the spirit of the template.
The examples config files are slightly changed so it's possible to run it in parallel without any problem

### CODE REVIEW

File: examples/modules/poisson/examples/red/red.geo [hunk 1]
```diff
@@ -1,10 +1,10 @@
 h = 0.1;
 
-Lx = DefineNumber[480*10^(-9), Name "x Lenght"];
-Ly = DefineNumber[320*10^(-9), Name "y Lenght"];
-Lz = DefineNumber[320*10^(-9), Name "z Lenght"];
-Lj = DefineNumber[160*10^(-9), Name "j Lenght"];
-t0x = DefineNumber[10*10^(-9), Name "Gate Lenght"];
+Lx = DefineNumber[480*10^(-6), Name "x Lenght"];
```
Comment (prudhomm) [inline examples/modules/poisson/examples/red/red.geo]: Lenght -> Length

File: examples/modules/poisson/pages/red/README.adoc [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+<button class="btn" data-clipboard-text="github:{repo:toolbox,path:examples/modules/poisson/examples/red}">
+Copy case option to clipboard
+</button>
+++++
+
+== Data files
+
+The case data files are available in Github link:{uri-data}/rotating-winch/[here]
+
+* link:{uri-data}/rotating-winch/biele.cfg[CFG file] - [link:{uri-data-edit}/red/red.cfg[Edit the file]]
```
Comment (prudhomm) [inline examples/modules/poisson/pages/red/README.adoc]: be careful, this is a copy past, you need to adapt to your case

File: examples/modules/poisson/pages/red/README.adoc [hunk 3]
```diff
@@ -1,12 +1,41 @@
...
+* link:{uri-data}/rotating-winch/biele.json[JSON file] - [link:{uri-data-edit}/red/red.json[Edit the file]]
```
Comment (prudhomm) [inline examples/modules/poisson/pages/red/README.adoc]: idem

File: examples/modules/poisson/pages/red/README.adoc [hunk 4]
```diff
@@ -24,6 +53,10 @@ The geometry has been generated with GMSH:
 include::{examplesdir}/red/red.geo[]
 ----
 
+
+N.B.: the unit of measure used in the `.geo` file are in _mm_.
```
Comment (prudhomm) [inline examples/modules/poisson/pages/red/README.adoc]: replace N.B. by `IMPORTANT:`

File: examples/modules/poisson/pages/red/README.adoc [hunk 5]
```diff
@@ -154,29 +187,15 @@ NOTE: Click top left button on opengl window to change basic visualisation featu
...
-To reproduce those results, one should run:
-[source,shell]
-----
-mpirun -np 4 feelpp_toolbox_mixed-poisson-model_3DP1 --config-file red.cfg
-mpirun -np 4 feelpp_toolbox_mixed-poisson-model_3DP1 --case "github:{repo:toolbox,path:examples/modules/poisson/examples/red}"
-----
+stem:[V^{ideal}_T = 0.8591] V, which agrees with the value predicted by the numerical simulation.
+
 
 == Bibliography
```
Comment (prudhomm) [inline examples/modules/poisson/pages/red/README.adoc]: any bilbiography on this?

**Comment by lsala** [comment]
I didn't know there were comments, I haven't received anything at the time, I thought everything was ok ...

**Comment by lsala** [comment]
Oh I had already fixed everything except the first one, the name in the geo file

**Pull request merged**